testing/str_test.py

```diff
@@ -12,7 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# pylint: disable=redefined-outer-name
+# pylint: disable=redefined-outer-name,W0703
+
+import sys
+
+
+def checkequal(expected, s, attr, *args):
+  assert getattr(s, attr)(*args) == expected
+
+
+def checkraises(expected, s, attr, *args):
```

Comment:
_checkraises() since this is a private helper function.

Also, I'd recommend matching the semantics of assertRaises as closely as possible in the unittest library since that may be familiar to people reading the code.